### PR TITLE
Fix hardcoded uplink

### PR DESCRIPTION
**Pull request by MggMuggins**
Per the discussion in #277

It feels a little silly to do a bunch of relatively more complex stuff at runtime to fix something that can be fixed by a single source of truth at compile time.

### CODE REVIEW

File: microcloud/service/lxd_config.go [hunk 1]
```diff
@@ -93,7 +96,7 @@ func (s LXDService) DefaultOVNNetwork(ipv4Gateway string, ipv4Range string, ipv6
 	}
 
 	ovnNetwork := api.NetworksPost{
-		NetworkPut: api.NetworkPut{Config: map[string]string{"network": "UPLINK"}, Description: "Default OVN network"},
+		NetworkPut: api.NetworkPut{Config: map[string]string{"network": DefaultOVNNetwork}, Description: "Default OVN network"},
 		Name:       "default",
```
Comment (masnax) [inline microcloud/service/lxd_config.go:100]: Given that we also have `"default"` as an `ovn` type network, I think the const name might be a bit misleading.

How about using `DefaultOVNNetwork` for `"default"` and `OVNUplinkNetwork` for `"UPLINK"`?

**Review by masnax — CHANGES_REQUESTED** [review]
Just one nit :)

File: microcloud/service/lxd_config.go (hunk 1 above)
Comment (tomponline) [reply microcloud/service/lxd_config.go:100]: I strongly agree. The uplink isn't an ovn network

**Comment by masnax** [comment]
@MggMuggins Heads up that this needs a rebase now.

**Comment by MggMuggins** [comment]
Woops; forgot about this one. Done. Test failure feels very unlikely to be related.

**Review by masnax — APPROVED** [review]

**Pull request merged**